### DISCUSSION TITLE
An overview of available directives for the Interactivity API

### DISCUSSION THREAD

**Discussion by ryanwelcher** [WordPress/developer-blog-content#285] [Topic Ideas]
As discussed in the [July 4th developer blog editorial meeting](https://wordpress.slack.com/archives/C03RL47B3S8/p1720098025766779), we will be creating an MVP for video on the developer blog.

This video will be on the available directives for the iAPI and will follow the [list of directives](https://developer.wordpress.org/block-editor/reference-guides/interactivity-api/api-reference/#list-of-directives) in the documentation showing a real-world example for each.

**Comment by zzap** [comment]
@ryanwelcher please ping me when the draft is ready for the first review. Thanks

**Comment by bph** [comment]
This topic was approved. Please continue conversation on the issue 
- #291
